indico/modules/rb/client/js/modules/bookRoom/actions.js

```diff
@@ -18,7 +18,8 @@ import {validateFilters} from '../../common/filters';
 import {preProcessParameters} from '../../util';
 import {actions as modalActions} from '../../modals';
 
-import {roomSearchActionsFactory, ajaxRules as ajaxFilterRules} from '../../common/roomSearch';
+import {roomSearchActionsFactory} from '../../common/roomSearch/actions';
+import {ajax as ajaxFilterRules} from '../../common/roomSearch/serializers';
```

Comment:
Same here.

Reply:
The thing is not all the imports at `roomSearch/index` resolve to a circular dependency. The only ones that are very deeply linked are `roomSearch/index -> importing -> actions/reducers` with `bookRoom/index -> importing -> actions/reducers`. 
Which means that we can still use the index on other occasions.

I agree that this becomes unconventional because some are direct imports, others go through the index so doing that change is one option.

However, in the case below of `rooms/selectors` and `user/selectors`, multiple imports depend on that, breaking them out of index would result in a lot of file changes.

Nonetheless, since this seems like an isolated case, what if we kept the imports and just ignored the rule here?

Reply:
I don't know. I agree that if the "indirect" (shorter) import works OK in other places, we can keep it. OTOH, does it really hurt changing all occurrences to the direct one? Let's maybe agree on a standard? @ThiefMaster what do you think?

Reply:
I guess being consistent is better - and it's a one-time change so even if it affects more files, it's probably the Right Thing ™ to do...

And since it's only within roombooking it should be easily testable too and thus be a rather safe change.

Reply:
Just as a quick comparison, before going into erradicating all indexes, check my latest commit.

Reply:
hm am i misunderstanding something? this commit looks more like the reverse of what i thought you'd do. or is it just to demonstrate the reverse of what you'd apply to the other files?

Reply:
I don't think the point would be to eradicate the indices, I would only do it for these particular exports...